Fix API get request to timemanager

Pinchett-i opened the pull request.

Pinchett-i committed:
Fix API get request to timemanager

RoelandMatthijssens commented on app.mjs:
```diff
@@ -54,7 +54,7 @@ app.use(function (err, req, res, next) {
 });
 
-const port = 3000
+const port = 3001
```
you probably don't want this as a hardcoded value

RoelandMatthijssens commented on config/default.json:
```diff
@@ -10,6 +10,9 @@
     "host": "timemanager-staging.blue-planet.be",
-    "api_key": "placeholder_key"
+    "headers": {
+      "AUTHORIZATION": "placeholder_key"
```
Where will you put the actual key eventually when deploying this to staging/production?

Pinchett-i replied:
I'm not sure, what I know is that I didn't want it on github. Moreover, the internet want to suggest ENV variables, but you already told me you disliked that approach, so I'm researching the other options

Pinchett-i committed:
Add Environment variables to override config

This commit allows the overriding of default configuration by using
environment variables (mainly to avoid storing password and keys).

Values can be defined in a `.env/<NODE_ENV>.env` file (default is
`developement`)
